packages/fxa-profile-server/lib/img-workers.js

```diff
@@ -15,22 +15,32 @@ const WORKER_URL = config.get('worker.url');
 const ACCEPT_ENCODING_HEADER = 'identity';
 
 exports.upload = function upload(id, payload, headers) {
-  return new P(function (resolve, reject) {
+  return new P(function(resolve, reject) {
     var url = WORKER_URL + '/a/' + hex(id);
 
+    const header_excludes = config.get('worker.headers_exclude');
+    const worker_headers = {...headers};
+    if (header_excludes.length) {
```

Comment:
This all looks good, and I can merge as it is.  But in the name of even fewer lines of code, we don't need this `if` since `for...of` can handle an empty array gracefully.

Let me know if I should merge as it is, or if you push a change, I'll merge after that.  Thanks!

Reply:
OK, removed the `if` here.